data_dictionary/storage_options.hh

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
> but a key in sstables regustly, but fixing it is beyond the scope of

Typo: regustly

Normally I don't get hang up on typos, but here it took my a while to figure out what this word wants to be.

Reply:
:scream: